service/dap/server_test.go

```diff
@@ -2377,44 +2377,37 @@ func TestLaunchRequestDefaults(t *testing.T) {
 
 func TestLaunchRequestDefaultsNoDebug(t *testing.T) {
```

Comment:
Oh, and please remove "Defaults" from the name. Just like the duplicate tests below that you removed, it's an artifact from copying another test that tested for various defaults of the args in the launch arguments.

Reply:
Done